src/main/java/org/ays/auth/model/AysPermissionDetail.java

```diff
@@ -0,0 +1,21 @@
+package org.ays.auth.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.ays.auth.model.enums.AysPermissionCategory;
+
+/**
+ * Represents a permission detail.
+ * It encapsulates information detail about a specific permission granted within the system.
+ */
+@Getter
+@Setter
+@SuperBuilder
+public class AysPermissionDetail {
```

Comment:
```suggestion
public class AysPermission extends BaseDomainModel {
```

Reply:
AysPermission sınıfı şu anda mevcut ancak dönülecek response'ta permissions alanının içerisinde BaseDomainModel ile gelen bilgiler istenmediği için, o sınıfı güncellemek yerine AysPermissionDetail sınıfını oluşturdum ve BaseDomainModel'ı extend etmedim

Response:
"response": {
    "createdUser": "ays-admin",
    "createdAt": "2023-05-07T09:45:08",
    "updatedUser": null,
    "updatedAt": null,
    "id": "0fe8ed42-bc8f-4e58-a0d1-7ad668c89b86",
    "name": "Kurum Yöneticisi",
    "status": "ACTIVE",
    "permissions": [
      {
        "id": "0418652d-d17c-41ed-b680-b262f451b2b9",
        "name": "role:list",
        "category": "ROLE_MANAGEMENT"
      }
    ]
  }